userguide/advanceduse/dav.rst

```diff
@@ -10,26 +10,78 @@ At the moment, there is no carddav implementation directly accessible from the U
 1) Follow this `guide <https://docs.ubports.com/en/latest/userguide/advanceduse/adb.html>`_ to activate Developer Mode and ADB (or SSH) connection.
 
 
-2) Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
+2) Download this `project <https://github.com/UT-ilities/UTouch_CalDAV_and_CardDAV_synchronization>`_ onto the device.
+    ``git clone https://github.com/UT-ilities/UTouch_CalDAV_and_CardDAV_synchronization.git``
+    
+    ``git checkout v0.1.0``
+    
+    ``# or``
+    
+    ``wget --quiet --show-progress https://github.com/UT-ilities/UTouch_CalDAV_and_CardDAV_synchronization/archive/v0.1.0.zip``
+    
+    ``unzip v0.1.0.zip``
 
-* server side : CAL_URL, **CONTACTS_URL**, USERNAME, PASSWORD (of your ownCloud/nextCloud/baikal/SOGO/... server)
-* CONTACT and CALENDAR _ NAME / VISUAL_NAME / CONFIG_NAME (it's more cosmetic)
-* CRON_FREQUENCY (for the frequency of synchronization)
-* Line 61: write ``sudo sh -c "echo '$COMMAND_LINE' > /sbin/sogosync"`` , instead of ``sudo echo "$COMMAND_LINE" > /sbin/sogosync``, to avoid permission denied error
+3) Modify or create a configuration file.
+    ``cp config-nextcloud-template.txt config-personal.txt``
+    
+    ``vim config-personal.txt``
+ 
+4) Edit or create the following configuration variables.
+    These three variable will be used for all calendars and contacts in configuration file
+  
+        * ``USERNAME`` # Credential presented to the server  
```

Comment:
Trying to `# comment` the end of these entries is not necessary. I'd prefer if you used definition lists, like this:

```
``USERNAME``
  Your username on the Cal/CardDAV server
``PASSWORD``
  Your password on the Cal/CardDAV server
```

See [Definition Lists in Quick reStructuredText](http://docutils.sourceforge.net/docs/user/rst/quickref.html#definition-lists) for more information.